bifrost.go

```diff
@@ -108,3 +109,15 @@ func newGoogleCloudStorage(g *BridgeConfig) (rainbowBridge, error) {
 		PublicRead:      g.PublicRead,
 	}, nil
 }
+
+// newGoogleCloudStorage returns a new client for AWS S3
+func newAmazonWebServices(g *BridgeConfig) (rainbowBridge, error) {
```

Comment:
I think we need to create an actual `S3` client here. Something with a type of say `s3.Client` or `s3.S3` that'll be reusable until disconnected else, the disconnect method won't be passing the `nil` check in the if block.